packages/calypso-e2e/src/lib/pages/themes-customizer-page.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import { BaseContainer } from '../base-container';
+
+/**
+ * Type dependencies
+ */
+import { Frame } from 'playwright';
+
+const selectors = {
+	frame: 'iframe[title="Customizer"]',
+};
+
+/**
+ * Component representing the Appearance > Customize page.
+ *
+ * @augments {BaseContainer}
+ */
+export class ThemesCustomizerPage extends BaseContainer {
+	frame!: Frame;
+
+	/**
+	 * Post-initialization steps.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async _postInit(): Promise< void > {
+		const elementHandle = await this.page.waitForSelector( selectors.frame );
+		this.frame = ( await elementHandle.contentFrame() ) as Frame;
+	}
+
+	/**
+	 * Closes the Appearance > Customize page when launched in a new tab.
+	 *
+	 * This behavior occurs when the customization option is clicked on from the
+	 * themes page after the particular theme has been applied.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async closeTab(): Promise< void > {
+		await this.page.close();
```

Comment:
This is the Customizer page, and since we aren't doing anything with this page at this point in time, this class is quite basic.

Reply:
What do you think of potentially renaming this class/file to be the `ThemesCustomizerPageWrapper` or something like that?

Because I believe if you go Appearance > Customize (like you say in the comments), there actually is _not_ an iframe - we only use that iframe in the popout tab that launches when you go from the theme selector.

That way, we could differentiate this wrapper holding that iframe from actual Customizer page content itself (which will probably have a lot of different actions on it actually related to customizing).

Reply:
This is some hard hitting investigative journalism here!

As I mentioned in the comment thread above - I had wrongly assumed that every instance of this customizer would be wrapped in an iFrame. This is totally my oversight and a bad assumption to boot, so thank you for investigating this.

I did think a bit over lunch on how best to go forward on this.

Since the presence of this object isn't really a critical piece of the actual spec, I am inclined to remove this class over renaming this to a wrapper. This would require modifying the test to wait on a popup, wait for it to load then perhaps check the title, all in the spec itself.

What do you think? I am not opposed to keeping the wrapper class with a new name either :)

Reply:
Yes, I really like the suggestion of removing the class, great idea! It's a pretty thin wrapper we have to interact with, and I think interacting with that pop-up tab and iframe directly, if needed, is very reasonably within the purview of a test script as it is needed.